config/routes.rb

```diff
@@ -22,6 +22,15 @@
     get "/delete", action: :confirm_destroy
   end
 
+  scope :projects do
```

Comment:
Tried to look at how we could nest this inside the `resources :projects` block, and this works, but I'm not sure if it's correct because it's not really a collection.

```ruby
resources :projects, only: %i[index show new create] do
  collection do
    get "completed"
  end

  collection do
    namespace :conversion do
      namespace :involuntary do
        get "new", to: "projects#new"
        post "new", to: "projects#create"
      end
    end
  end
```

Reply:
I think the way we have it is correct, as we're not using a `project` resource. The `scope` syntax works because we want to have `/projects/` on the URL but the controller folders aren't nested inside `projects`. I *think* that's the intention of scopes in routes.

Reply:
👍